### ISSUE TITLE
[Accessibility]:

### ISSUE THREAD

**Issue by amortiz** [carbon-design-system/carbon-charts#2040]
### Application/Team

IBM Cloud infrastructure

### Environment

Browser

### What happened? What did you expect to happen instead?

On any chart, when you click "More option", and then you click escape to close the overflow menu, focus is lost. It should re-focus the "More option" button.

Full AVT report:

```
Component: More option

Issue description: When we press escape from the More opation , Focus is lost.

Recommended fix: When we press more option button and we press escape focus is lost. it should focus on More options again.

User impact: Users, especially those relying on keyboard navigation or assistive technologies, may find it challenging to reorient themselves on the page and continue their tasks.

WCAG Mapping: 2.4.3 Focus Order

Level of Compliance: A

Severity: Low

Test tool: JAWS_Chrome

```

### What WCAG 2.1 checkpoint does the issue violate?

 2.4.3 Focus Order

### Version

Latest, just try any chart

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P3
